doc/level1.rst

```diff
@@ -275,6 +275,23 @@ above rules, such headers will be judged invalid, and discarded by
 level 1 MUAs.  Such an update to the specification will also have
 to describe how an updated MUA will deal with multiple valid headers.
 
+A MUA MUST NOT send Autocrypt headers above 10 KiB in size
+(including the Autocrypt: prefix and folding whitespace).
+Conversely, MUAs SHOULD not process Autocrypt headers
+larger than 10 KiB in size.
+
+.. note::
+
+  - An Autocrypt header following this specification
+    will be up to 3 KiB large.
+    Some users import 4096-bit RSA key configurations,
```

Comment:
itym "Some users *might* import" -- and i'm not sure whether "configurations" is the right term.  can we just say "4096-bit RSA keys"

Reply:
similarly resolved but not changed nor commented? @azul I'm confused by your understanding of "resolve" :)

Reply:
sorry... had forgotten to push. Should now actually be resolved.